api/v1/controllers/samples.js

```diff
@@ -150,13 +151,25 @@ module.exports = {
     const resultObj = { reqStartTime: new Date() };
     const sampleQueryBody = req.swagger.params.queryBody.value;
 
-    return u.getUserNameFromToken(req,
-      featureToggles.isFeatureEnabled('enforceWritePermission'))
-    .then((userName) => {
+    return authUtils.getUser(req)
+    .then((user) => {
```

Comment:
if request is sent without authentication token, request will still be granted. The createdBy field will be stripped out because of its null value, so the response will NOT contain createdBy, user fields.